tests/system/backward_compatiblity/test_code_executed_in_runing_job.py

```diff
@@ -0,0 +1,30 @@
+import pathlib
+
+import pytest
+
+import mlrun
+from tests.system.base import TestMLRunSystem
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+class TestCodeExecutedInRunningJob(TestMLRunSystem):
```

Comment:
I suggest to change the hierarchy to be:
module name: `backward_compatibility` (no change)
test class name: `TestAPI` (can also be `TestAPIBackwardCompatibility`
test name: `test_endpoints_called_by_sdk_from_inside_jobs`